src/python/pants/build_graph/app_base.py

```diff
@@ -81,6 +81,21 @@ def filemap(self):
         filemap[abspath] = self.mapper(abspath)
     return filemap
 
+  @memoized_property
+  def relative_filemap(self):
```

Comment:
Let's extract one common function here, which both properties can call, rather than copying and pasting the code :)

Reply:
@jsirois is this part of a public API, or can we just turn this into a `@memoize`'d function with an argument?